kordac/tests/TableOfContentsTest.py

```diff
@@ -7,7 +7,15 @@
 
 
 class TableOfContentsTest(ProcessorTest):
+    '''The table-of-contents processor inherits from the generic
+    tag procesor.  The tests contained here test that arguments
+    and the output (html-template) work as expected.
+    '''
```

Comment:
the testing on this processor just seem way too light for a processor that generates potentially a large amount of html output...